src/int256.rs

```diff
@@ -295,3 +319,61 @@ fn check_from_str_radix_overflow_underflow() {
     let val = Int256::from_str_radix(super_small, 10);
     assert!(val.is_err());
 }
+
```

Comment:
I noticed some similar tests in the now-defunct num_conversion.rs code in gravity_utils. I feel that the tests are more relevant in this upstream crate so I have expanded on to them here and in uint256.rs.

test_to_primitive_64() passes with or without the above changes to int256.rs